### DISCUSSION TITLE
How to intstall?? Как установить?

### DISCUSSION THREAD

**Discussion by ghost** [DavidRomanovizc/DatingBot#58] [General]
Could you describe in detail what to do and how to install and run!!

**Comment by dzhalaevd** [comment]
> Could you describe in detail what to do and how to install and run!!

Hello please read the README file, if your problem will not solve, text me in telegram 
https://t.me/mytgpprofile

**Reply by ghost** [reply]
https://t.me/SENATOR_KAYNE you can write me please because i have a ban in tg write pls
